docs/reference/commandline/service_create.md

```diff
@@ -758,7 +758,7 @@ The swarm extends my-network to each node running the service.
 Containers on the same network can access each other using
 [service discovery](https://docs.docker.com/engine/swarm/networking/#use-swarm-mode-service-discovery).
 
-Long form syntax of `--network` allows to specify list of aliases and driver options:  
+Long form syntax of `--network` allows to specify list of aliases and driver options:
```

Comment:
Please exclude this file from the commit. It only contains unrelated formatting changes.

Reply:
Fixed.